projects/managed-hubs/incidents.md

```diff
@@ -0,0 +1,107 @@
+
+   We've also opened an Incident Report in this issue where you can track progress if you wish: { LINK TO INCIDENT REPORT }.
+ 
+   We'll prioritize resolving this incident over our other work, and
+   will communicate with you throughout our attempts to resolve it.
+   We might be in touch with requests for clarifications if needed.
+   ```
+2. **Open an incident issue**.
+   For each {term}`Incident` we create a dedicated issue to track its progress. [{bdg-primary}`open an incident issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+Hub+Incident%2Csupport&template=3_incident-report.md&title=%5BIncident%5D+%7B%7B+TITLE+%7D%7D) and notify our engineering team via Slack.
+3. **If you can resolve in 30 minutes**, try doing so.
+4. **If you cannot resolve in 30 minutes**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
+5. **Designate an {term}`Incident Commander`**. If the Support Steward wishes to designate another team member as Incident Commander, should do so in the Incident issue.
+6. **Investigate and resolve the incident**. The Incident Commander should follow the structure of the incident issue opened in the step above.
+7. **Delegate to Subject Matter Experts as-needed**. The Incident Commander is empowered to delegate actions to Subject Matter Experts in order to investigate and resolve the incident quickly.
+8. **Communicate every few hours**. The {term}`Incident Commander` is expected to communicate incident status and plan with the {term}`Support Steward`s, are are expected to communicate to the {term}`Community Representative`s. Provide periodic updates to communities as we attempt to resolve the incident. Here is a template to get started:
+
+   ```
+   Hello { NAME }, this is a quick update on our progress resolving
+   your incident.
+
+   We believe the problem is { XXX } and are investigating { YYY }
+   to resolve it. We will keep you updated as we continue to make progress.
+   Please let us know if you have had more reports of issues,
+   or reports that your issues have gone away.
+   ```
+9. **Communicate when the incident is resolved**. When we believe the incident is resolved, communicate with the Community Representative that things should be back to normal. Mark the FreshDesk ticket as {guilabel}`Resolved`.
+10. **Fill in the {term}`Incident Report`**. The Incident Commander should do this in partnership with the Incident Response Team.
+11. **Close the incident ticket**. Once we have confirmation from the community (or no response after 48 working hours), and have filled in the incident {term}`Incident Report`, then close the incident by:
+    - Closing the incident issue on GitHub
+    - Marking the FreshDesk ticket as {guilabel}`Closed`
+
+## Key terms
```

Comment:
```suggestion

## Handing off Incident Commander status

During an incident, it may be necessary to designate another person to be the Incident Commander.
For example, if it is getting late in the current IC's time zone, they feel burnt out from leading the incident response, or there is someone with better visibility or experience to be the Incident Commander.
This is encouraged and expected, especially for more complex or longer incidents!

To designate another team member as the Incident Commander, follow these steps:

1. **Confirm with them** that they are able and willing to serve as the Incident Commander
2. **Update the Incident Report issue** by updating the Incident Commander name in the top comment.
3. **Notify the team** with a comment in the Incident Report issue.

## Key terms
```